drivers/spi/spi_stm32.h

```diff
@@ -1,26 +1,49 @@
 /*
- * Copyright (c) 2016 BayLibre, SAS
+ * Copyright (c) 2018 Bobby Noelte
```

Comment:
when rebasing, you'll probably modify spi_ll_stm32 driver (do not create a new one), and thus you'll keep the copyright accordingly

Reply:
I replaced the original file totally. No BayLibre IP was left. Anyway the header file is now gone.